Tests/UITests/TestApp/ContentView.swift

```diff
@@ -48,8 +49,10 @@ struct ContentView: View {
         Button("Request Notification Permissions") {
             _Concurrency.Task {
                 try await scheduler.requestLocalNotificationAuthorization()
+                notificationAuthorizationGranted = await scheduler.localNotificationAuthorization
             }
         }
+        .disabled(notificationAuthorizationGranted)
```

Comment:
It would probably make sense to use the flag of the `TestAppScheduler` here instead of a local state property.

Reply:
Sadly, I think this isn't possible as the `localNotificationAuthorization` property from the `TestAppScheduler` is an `async` property and the `.disabled()` view modifier doesn't allow `async` properties, just regular ones.

The workaround that is currently implemented `await`'s on the `localNotificationAuthorization` property after `requestLocalNotificationAuthorization()` function is called. The result of that call is then stored in a State variable within the view.

Reply:
Makes sense, thank you for the context!